ghost/core/core/server/update-check.js

```diff
@@ -14,14 +14,22 @@ const UpdateCheckService = require('@tryghost/update-check-service');
  * Initializes and triggers update check
  * @param {Object} [options]
  * @param {Boolean} [options.rethrowErrors] - if true, errors will be thrown instead of logged
+ * @param {Boolean} [options.forceUpdate] - if true, the update check will be triggered regardless of the environment or scheudle, defaults to config if no value provided
+ * @param {String} [options.updateCheckUrl] - the url to check for updates against, defaults to config if no value provided
  * @returns {Promise<any>}
  */
-module.exports = async ({rethrowErrors = false} = {}) => {
-    const allowedCheckEnvironments = ['development', 'production'];
+module.exports = async ({
+    rethrowErrors = false,
```

Comment:
Nice. Also gets us a step closer to passing all the configs as a parameters instead of depending on the config module 👌